query-node/mappings/src/content/nft.ts

```diff
@@ -930,3 +945,44 @@ export async function contentNft_BuyNowCanceled({ event, store }: EventContext &
 
   await store.save<BuyNowCanceledEvent>(announcingPeriodStartedEvent)
 }
+
+export async function contentNft_NftSlingedBackToTheOriginalArtist({
+  event,
+  store,
+}: EventContext & StoreContext): Promise<void> {
+  // common event processing
+
+  const [videoId, ownerId] = new Content.NftBoughtEvent(event).params
+
+  // load NFT
+  const { video, nft } = await getNftFromVideo(
+    store,
+    videoId.toString(),
+    'Non-existing video was slinged',
+    'Non-existing nft was slinged'
+  )
+
+  // load channel
+  const channel = (await getById(store, Video, video.getId(), ['channel'])).channel
+
+  // load channel ownerCuratorGroup (if any)
+  const ownerCuratorGroup = (await getById(store, Channel, channel.getId(), ['ownerCuratorGroup'])).ownerCuratorGroup
+
+  nft.ownerMember = undefined
+  nft.ownerCuratorGroup = ownerCuratorGroup
```

Comment:
I don't think there is any reason why the channel couldn't be owned by a member here, or is it?
```suggestion
  nft.ownerMember = video.channel?.ownerMember
  nft.ownerCuratorGroup = video.channel?.ownerCuratorGroup
```